docs/tips-and-tricks.rst

```diff
@@ -45,3 +45,35 @@ Note that currently it's not possible to pin an app to a commit, so if your
 flatpaks are updated either manually or automatically, the downgraded app will
 be included in the updates. See https://github.com/flatpak/flatpak/issues/3078
 
+Bisecting regressions in application builds
+-------------------------------------------
+
+In case the newest builds of an application introduce regressions, you can use
+``flatpak-bisect`` to discover which commit introduced the regression. It works
+just like ``git bisect``.
+
+First you update the application and get its history::
+
+  $ flatpak-bisect org.gnome.gedit start
+
+Then, you should set the current commit as the first bad commit::
+
+  $ flatpak-bisect org.gnome.gedit bad
+
+Now you need to find the hash of the first known good commit::
```

Comment:
Should we mention `flatpak-bisect log` here?

Reply:
sure.